### ISSUE TITLE
[REC] Casting Lightning Bolt or Chain Lightning prevents actions which check Lightning Rod debuff

### ISSUE THREAD

**Issue by mwojtkowski** [Hekili/hekili#3868]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

Shaman - Elemental

### Describe the Issue

Both Lightning Bolt and Chain Lightning have: 
``` if talent.lightning_rod.enabled then applyDebuff( "target", "lightning_rod" ) end ```
which should be removed

only Tempest, Earth Shock, Elemental Blast, and Earthquake  can apply that debuff to the target

Because of above if lightning rod debuff is missing on current target but we are casting lightning bolt or chain lightning any action that checks if lightning rod debuff is missing will fail until end of that cast

Action List single_target, Entry 24 Earth Shock
```
( ( buff.master_of_the_elements.up | lightning_rod = 0 ) 
& cooldown.stormkeeper.remains > 10 
& ( rolling_thunder.next_tick > 5 | ! talent.rolling_thunder.enabled ) | buff.stormkeeper.up ) 
& talent.storm_elemental.enabled 
& spell_targets.chain_lightning = 1
```
above action will not be recommended while casting LB or CL even if target has not lightning rod debuff

### How to Reproduce

1. Enter game as Elemental shaman
2. Use Single Target build from wowhear (Farseer)
3. When Hekili will suggest Earth Shock start casting Lightning Bolt or Chain Lightning
4. Earth Shock Icon will disappear until you finish your cast

### Snapshot (Link)

https://pastebin.com/Yj5TWBEB

**Comment by syrifgit** [comment]
I see, the issue is that there is a difference between Elemental and Enhance as to which spells trigger it.

**Comment by Hekili** [comment]
Resolved in 1.0.17a.
